### ISSUE TITLE
Fix velocity depth average.

### ISSUE THREAD

**Issue by bangerth** [geodynamics/aspect#588]
As noted in #569, the velocity depth average actually computes the square of what it's supposed to do:
```
    template <int dim>
    class FunctorDepthAverageVelocityMagnitude
    {
      public:
        FunctorDepthAverageVelocityMagnitude(const FEValuesExtractors::Vector &field)
          : field_(field) {}

        bool need_material_properties() const
        {
          return false;
        }

        void setup(const unsigned int q_points)
        {
          velocity_values.resize(q_points);
        }

        void operator()(const MaterialModel::MaterialModelInputs<dim> &,
                        const MaterialModel::MaterialModelOutputs<dim> &,
                        FEValues<dim> &fe_values,
                        const LinearAlgebra::BlockVector &solution,
                        std::vector<double> &output)
        {
          fe_values[field_].get_function_values (solution, velocity_values);
          for (unsigned int q=0; q<output.size(); ++q)
            output[q] = velocity_values[q] * velocity_values[q];
        }
```
Fix this.

**Comment by bangerth** [comment]
There's also the opposite case to #589: it doesn't respect the `year_in_seconds` setting.

**Comment by bangerth** [comment]
Fixed by #592.
